tracing/tracing.go

```diff
@@ -65,16 +94,30 @@ func WithEndpointURL(url string) StartOption {
 	}
 }
 
+// WithGlobalTag sets a key/value pair which will be set as a tag on all spans
+// created by tracer. This option may be used multiple times.
+// Note: Since the underlying transport is Zipkin, only values with strings
+// are accepted.
+func WithGlobalTag(k string, v string) StartOption {
+	return func(c *config) {
+		globalTag := tracer.WithGlobalTag(k, v)
+		c.globalTags = append(c.globalTags, globalTag)
+	}
+}
+
 // Start tracing globally
 func Start(opts ...StartOption) {
 	c := defaultConfig()
 	for _, fn := range opts {
 		fn(c)
 	}
 
+	startOptions := append(c.globalTags, tracer.WithServiceName(c.serviceName))
+	startOptions = append(startOptions, tracer.WithZipkin(c.serviceName, c.url, c.accessToken))
 	tracer.Start(
-		tracer.WithServiceName(c.serviceName),
-		tracer.WithZipkin(c.serviceName, c.url, c.accessToken))
+		startOptions...
```

Comment:
Why not keep the two previous lines and then add another one with `c.globalTags...`? This way the appends above can be eliminated.

Reply:
The compiler didn't like it this is the error I got 
`./tracing.go:91:14: too many arguments in call to tracer.Start
        have (tracer.StartOption, tracer.StartOption, []tracer.StartOption...)
        want (...tracer.StartOption)`

Reply:
oh, makes sense thinking about it: the parameters are going to be converted to an array so this looks like a different signature...